app/components/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+
+export default function Header() {
+  return (
+    <>
+      <header className="flex flex-row justify-between bg-pink">
+        <div className="flex flex-row justify-around w-1/2">
+          <button>Menu</button>
+          <button>Specials</button>
+          <button>Location</button>
+        </div>
+        <div className="flex flex-row justify-center w-1/2">
+          <button>Start Order Here!</button>
+          <div className="h-1/4 w-1/4 mx-6">
+            <Image
+              src={"/misc/chillNoBG.png"}
+              // TDL- want to figure out what needs to happen for fill
+              width={50}
+              height={50}
+              alt=""
```

Comment:
Remember to always fill out your alt tags. This may seem tedious, but it's better to get it done upfront than to do 50 of them all at once later.